### PR TITLE
feat(selector): add multi select support

### PR DESCRIPTION
**Pull request by amcdnl**
Adds support for multiple selects #51

### CODE REVIEW

File: packages/store/src/store.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class Store {
   /**
    * Selects a slice of data from the store.
    */
-  select<T>(selector: (state: any) => T): Observable<T>;
+  select<T>(selector: (state: any, states?: any[]) => T): Observable<T>;
```
Comment (markwhitfeld) [inline packages/store/src/store.ts]: This does not seem to match the signature  executed by the selector above.
Shouldn't it rather be: `selector: (state: any, ...states?: any[]) => T)`

File: packages/store/src/store.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ export class Store {
   /**
    * Select one slice of data from the store.
    */
-  selectOnce<T>(selector: (state: any) => T): Observable<T>;
+  selectOnce<T>(selector: (state: any, states?: any[]) => T): Observable<T>;
```
Comment (markwhitfeld) [inline packages/store/src/store.ts]: Same comment as above.

File: packages/store/src/store.ts [hunk 3]
```diff
@@ -56,7 +56,7 @@ export class Store {
   /**
    * Select a snapshot from the state.
    */
-  selectSnapshot<T>(selector: (state: any) => T): T {
+  selectSnapshot<T>(selector: (state: any, states?: any[]) => T): T {
```
Comment (markwhitfeld) [inline packages/store/src/store.ts]: Same comment as above.

File: packages/store/src/store.ts (hunk 1 above)
Comment (amcdnl) [reply packages/store/src/store.ts]: #facepalm

**Pull request merged**